composer.json

```diff
@@ -86,7 +93,6 @@
         ],
         "cs:check": "@php php-cs-fixer check --verbose --diff",
         "cs:fix": "@php php-cs-fixer fix",
-        "cs:fix:parallel": "echo '🔍 Will run in batches of 50 files.'; if [[ -f .php-cs-fixer.php ]]; then FIXER_CONFIG=.php-cs-fixer.php; else FIXER_CONFIG=.php-cs-fixer.dist.php; fi; php php-cs-fixer list-files --config=$FIXER_CONFIG | xargs -n 50 -P 8 php php-cs-fixer fix --config=$FIXER_CONFIG --path-mode intersection 2> /dev/null",
```

Comment:
you might consider output a warning and alias this command to `cs:fix`, so possibly existing scripts using it, will not break immediately after release

Reply:
It's not related to enabling parallel by default (for everyone) IMHO. But since this command is only for local development, and `.php-cs-fixer.dist.php` has `->setParallelConfig(ParallelConfig::detect())`, then it's almost not needed anymore. "Almost", because developers who have `.php-cs-fixer.php` in their project won't have parallel runner enabled without additional manual work.